snippets/language-html.cson

```diff
@@ -92,13 +92,13 @@
     'body': '<dfn>$1</dfn>$0'
   'Description List':
     'prefix': 'dl'
-    'body': '<dl id="$1">\n\t$2\n</dl>'
+    'body': '<dl class="$1">\n\t$2\n</dl>'
   'Definition Term':
     'prefix': 'dt'
     'body': '<dt>$1</dt>$0'
   'Div':
     'prefix': 'div'
-    'body': '<div${1: id="${2:name}"}>\n\t$3\n</div>'
+    'body': '<div${1: class="$2"}>\n\t$3\n</div>'
```

Comment:
@kevinsawicki What do you think about the lack of space after the `div` here? I'm not sure – it's tripped me up quite a lot in practice, and you probably shouldn't be creating plain `div`s without a class anyway.